api/v1/submissions/SubmissionHandler.inc.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+/**
+ * @file api/v1/submission/SubmissionHandler.inc.php
+ *
+ * Copyright (c) 2014-2016 Simon Fraser University Library
+ * Copyright (c) 2003-2016 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @class SubmissionHandler
+ * @ingroup api_v1_submission
+ *
+ * @brief Handle API requests for submission operations.
+ *
+ */
+
+import('lib.pkp.classes.handler.APIHandler');
+
+class SubmissionHandler extends APIHandler {
+
+	/**
+	 * The unique endpoint string for this handler
+	 *
+	 * @param string
+	 */
+	protected $_handlerPath = 'submissions';
```

Comment:
`$_handlerPath` property was already defined in APIHandler class; its value should be set in the constructor. 
Also `APIHandler::getEndpointPattern()` is not an abstract class and uses `$_ handlerPath` (which will be initialized by a subclass). In APIHandler::__constructor() , I suggest to check whether `$_handlerPath` was set and raise and exception if not.